src/platform/packages/private/kbn-index-editor/src/components/file_preview.tsx

```diff
@@ -54,117 +41,8 @@ const FILE_PREVIEW_ROWS_LIMIT = 10;
-          if (fileStatus.data) {
-            return previewDocs(
-              fileStatus.data,
-              fileStatus.results?.ingest_pipeline!,
-              FILE_PREVIEW_ROWS_LIMIT
-            );
-          }
-        })
-      );
-
-      // Create an ad-hoc data view for each file
-      const adHocDataViews = await Promise.all(
-        filesStatus.map((fileStatus) => {
-          return data.dataViews.create(
-            {
-              id: fileStatus.fileName,
-              title: `temp_${fileStatus.fileName}`,
-              allowNoIndex: true,
-            },
-            true,
-            false
-          );
-        })
-      );
-
-      if (!isMounted()) return;
-
-      setFilePreviewItems(
-        filesStatus.map<FilePreviewItem>((status, index) => {
-          const promisePreviewResult = previewResults[index];
-
-          const dV = adHocDataViews[index];
-
-          const columnNames = status.results?.column_names || [];
-          const mappings = status.results?.mappings || {
-            properties: {},
-          };
-
-          let fileClash: FileClash | undefined = uploadStatus.fileClashes[index];
-          if (fileClash.clash === CLASH_ERROR_TYPE.NONE) {
-            fileClash = undefined;
-          }
-
-          const item: FilePreviewItem = {
-            fileName: status.fileName,
-            dataView: dV,
-            columnNames,
-            mappings,
-            fileContents: status.fileContents
-              .split('\n')
-              .slice(0, FILE_PREVIEW_ROWS_LIMIT)
-              .join('\n'),
-            filePreview: {},
-            ...(fileClash ? { fileClash } : {}),
-          };
-
-          if (promisePreviewResult.status === 'fulfilled') {
-            const filePreview = promisePreviewResult.value!;
-            const validESHits: EsHitRecord[] = filePreview?.docs
-              ?.filter((d) => !!d.doc)
-              .map<EsHitRecord>((d) => d.doc as EsHitRecord);
-
-            const previewErrors: string[] = filePreview?.docs
-              ?.filter((d) => !!d.error)
-              .map((err) => {
-                return `${err.error?.reason} [${err.error?.caused_by?.reason}]`;
-              });
-
-            const dataRecords = validESHits?.map((doc, i) => {
-              return buildDataTableRecord(
-                {
-                  ...doc,
-                  _id: `${status.fileName}-${i}`,
-                  _index: `temp_index_${i}`,
-                },
-                dV
-              );
-            });
-
-            item.filePreview.sampleDocs = dataRecords;
-            item.filePreview.errors = previewErrors;
-          } else {
-            //
-          }
-
-          return item;
-        })
-      );
-    } catch (error) {
-      // Handle error appropriately, e.g., log it or show a notification
-    }
-  }, [data.dataViews, filesStatus, isMounted, previewDocs, uploadStatus.fileClashes]);
-
   useEffect(
     function fetchFilePreviewAfterAnalysis() {
```

Comment:
Probably we should update the name of this effect, as `fetch` might be interpreted as getting the preview from an API which is no longer the case, (we are not triggering the fetch anymore but it's already provided by the fileUploader) .
```suggestion
    function getFilePreviewAfterAnalysis() {
```
Same with some of the comments in this function which refers to fetching data.

Reply:
Name changed in https://github.com/elastic/kibana/pull/237679/commits/3c373335554f59398ff140df8ebab960d3445ee1